### PR TITLE
Add even a little more valuable data to the syslog

### PR DESCRIPTION
**Pull request by mdiehm**
Maybe some of this data is also wanted in the mail - esp. ratio number and prevmatch?

### CODE REVIEW

File: lib/Smokeping.pm [hunk 1]
```diff
@@ -1960,7 +1961,11 @@ ALERT
             } else {
                         do_debuglog("Alert \"$_\": no match for target $name\n");
             }
-            $tree->{'prevmatch'.$s}{$_} = $match;
+            if ($match == 0) {
+                $tree->{'prevmatch'.$s}{$_} = $match;
+            } else {
+                $tree->{'prevmatch'.$s}{$_} += $match;
+            }
         }
```
Comment (oetiker) [inline lib/Smokeping.pm]: what is the rational behind this change ?

**Comment by mdiehm** [comment]
Before:
Mon Jan 25 14:17:42 2016 - Alert onelossOver20percent is active for master-short.klb82-fwed-short-10-33-101-250

After:
Fri Jan 29 14:45:58 2016 - Alert onelossOver20percent is active for master-short.klb70-direct-short-131-152-6-229 loss: 100%(3/3) rtt: U prevmatch: 15 comment: loss 1st time

(prevmatch meaning how many times it has been matched before while being offline - if the host is back online prevmatch will be reset to 0)

File: lib/Smokeping.pm (hunk 1 above)
Comment (mdiehm) [inline lib/Smokeping.pm]: not worth it?
We find that info quite helpful while debugging network problems...

Comment (oetiker) [inline lib/Smokeping.pm]: sorry got distracted ...

**Pull request merged**